### DISCUSSION TITLE
genral setting page and minimize to taskbar

### DISCUSSION THREAD

**Discussion by charlesyu08** [EDCD/EDDI#2192] [Needs thought]
It would be great to have all app settings in on single tab, also it would be nice to able to minimize into the taskbar

**Comment by Tkael** [comment]
Please provide more detail on the settings you'd like to see included on a single page?
EDDI already minimizes to the Windows taskbar.

**Reply by charlesyu08** [reply]
I mean minimizing to the side of the taskbar like an icon
for the settings, I am thinking of the eddn responder, edsm responder, frontier api, inara responder can all be in a single page as you only need to set it up once to make it work

**Reply by Tkael** [reply]
Thanks for the feedback. 

Starting minimized is something that we've considered previously (ref. #231 and #1258), but doing so wouldn't be in line with Microsoft's guidance on the use of the system tray and we have chosen to respect that guidance. 

The EDDN responder, EDSM responder, Frontier API, and and Inara Responder are each separate and independent modules in the code. I don't think it's going to be feasible, at least for the foreseeable future, to combine them.
